netdata-installer.sh

```diff
@@ -934,6 +934,34 @@ if [ "$have_autotools" ]; then
   fi
 fi
 
+# function to extract values from the config file
+config_option() {
+  section="${1}"
+  key="${2}"
+  value="${3}"
+
+  if [ -s "${NETDATA_PREFIX}/etc/netdata/netdata.conf" ]; then
+    "${NETDATA_PREFIX}/usr/sbin/netdata" \
+      -c "${NETDATA_PREFIX}/etc/netdata/netdata.conf" \
+      -W get "${section}" "${key}" "${value}" ||
+      echo "${value}"
+  else
+    echo "${value}"
+  fi
+}
+
+# the user netdata will run as
+if [ "$(id -u)" = "0" ]; then
+  NETDATA_USER="$(config_option "global" "run as user" "netdata")"
+  ROOT_USER="root"
+else
+  NETDATA_USER="${USER}"
+  ROOT_USER="${USER}"
```

Comment:
I would argue that this variable should probably be renamed throughout the file to be a bit more descriptive of what it’s used for, but that’s also out of scope for this PR.

Reply:
I agree.

Reply:
Make a PR later please.